tests/sentry/utils/sdk_crashes/test_sdk_crash_detector.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from sentry.utils.sdk_crashes.sdk_crash_detector import SDKCrashDetector
+
+
+@pytest.mark.parametrize("field_containing_path", ["package", "module", "abs_path", "filename"])
+def test_build_sdk_crash_detection_configs(empty_cocoa_config, field_containing_path):
+
+    empty_cocoa_config.sdk_frame_config.path_patterns = {"Sentry**"}
```

Comment:
If configuring a field in the fixture is a common use case, the fixture could return a function that accepts a parameter and returns the full `SDKCrashDetectionConfig`.

Reply:
Currently, it is not, but if it turns into a common use case, I will use a function with a parameter. Thanks for pointing that out 😃 .